test-automation/tests/features/steps/residents_step_defs.py

```diff
@@ -17,9 +17,9 @@ def user_can_view_residents(self):
 @step(r"the user searches for a resident by \"(.*?)\"")
 def user_searches_for_resident(self, mode):
     if mode == "name":
-        assert search_for(mode, "Aava E") == True
+        assert search_for(mode, "Aaliyah S") == True
     elif mode == "home":
-        assert search_for(mode, "Lehmus") == True
+        assert search_for(mode, "West Alfonzo furt") == True
```

Comment:
These data are likely to change each time, if using the random data generator. It may be necessary to have a set-up step that inserts a resident and home, so they can be known.